packages/ppbgdi/data_stream/cloudfront_logs/fields/fields.yml

```diff
@@ -79,8 +79,7 @@
       type: keyword
       description: |-
         Kinesis type.
-- name: http.request.headers.*
+- name: http.request.header.*
   type: keyword
   description: Not yet but already in use ecs fields
```

Comment:
`Not yet *defined/approved* but already...`?

Reply:
Done